### ISSUE TITLE
[Error Fixed in Upcoming Release] Oops! Database Error: attempt to write a readonly database in a privileged proxmox container. 

### ISSUE THREAD

**Issue by auggy** [crocodilestick/Calibre-Web-Automated#46]
im running in a privileged proxmox container and using a .DB from a  working Calibre-Web-Automated 

I keep getting this error when uploading 
ERROR {cps.editbooks:302} Database error: (sqlite3.OperationalError) attempt to write a readonly database
[SQL: INSERT INTO authors (name, sort, link) VALUES (?, ?, ?)]

and this error when deleting 
Deleting book 67 failed: [Errno 13] Permission denied: '/calibre-main/Calibre Library

Hopefully i gave enough info

**Comment by crocodilestick** [comment]
Hey @auggy , this is a known issue with proxmox in V1.2.2 since proxmox uses a weird permission assignment system for containerised applications.

This should be fixed in the newest upcoming version V2.0.0 which you can try with the following docker compose until the official release:

~~~
--- services: calibre-web-automated: image: crocodilestick/calibre-web-automated:release-candidate container_name: cwa-v2.0.0-rc environment: - PUID=1000 - PGID=1000 - TZ=UTC - DOCKER_MODS=linuxserver/mods:universal-calibre volumes: - . :/config - . :/cwa-book-ingest - . :/calibre-library #- /path/to/where/you/keep/your/books:/books #Optional #- /path/to/your/gmail/credentials.json:/app/calibre-web/gmail.json #Optional ports: - 8084:8083 # Change the first number to change the port you want to access the Web UI, not the second restart: unless-stopped
~~~

**Comment by auggy** [comment]
that solution worked lovely, thanks
